data/channels.csv

```diff
@@ -1,4 +1,7 @@
 id,name,native_name,network,country,subdivision,city,broadcast_area,languages,categories,is_nsfw,launched,closed,replaced_by,website,logo
+EarthxTV.us,EarthxTVV,,,US,,,c/US,eng,,FALSE,,,,https://earthxtv.com/,https://video.earthxtv.com/themes/custom/earth/earthxtv/logo.png

```

Comment:
```suggestion
EarthxTV.us,EarthxTV,,,US,,,c/US,eng,,FALSE,,,,https://earthxtv.com/,https://video.earthxtv.com/themes/custom/earth/earthxtv/logo.png
```

Reply:
Thanks but It won't let me commit this, can you try please? Thanks